Display unique identifiers(Email Id) in add existing member functionality

krishna619 opened the pull request:
<!--
This section can be deleted after reading.

We employ the following branching strategy to simplify the development process and to ensure that only stable code is pushed to the `master` branch:

- `develop`: For unstable code: New features and bug fixes.
- `master`: Where the stable production ready code lies. Only security related bugs.

NOTE!!!

ONLY SUBMIT PRS AGAINST OUR `DEVELOP` BRANCH. THE DEFAULT IS `MAIN`, SO YOU WILL HAVE TO MODIFY THIS BEFORE SUBMITTING YOUR PR FOR REVIEW. PRS MADE AGAINST `MAIN` WILL BE CLOSED.
-->

<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**What kind of change does this PR introduce?**


<!-- E.g. a bugfix, feature, refactoring, etc… -->
feature - Displays unique identifiers(Email Id) in add existing member functionality


Fixes #1738 

**Did you add tests for your changes?**

<!--Yes or No. Note: Add unit tests or automation tests for your code.-->
No

**Snapshots/Videos:**

<!--Add snapshots or videos wherever possible.-->

**If relevant, did you update the documentation?**

<!--Add link to Talawa-Docs.-->

**Summary**

<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue for more information. -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**Other information**

<!--Add extra information about this PR here-->

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

<!--Yes or No-->

krishna619 commented:
@palisadoes please review.

dhanagopu commented:
@krishna619 Please fix the failing tests

dhanagopu commented:
@Kevoniat @beingnoble03 Please review this PR

beingnoble03 commented:
@krishna619 please add a screenshot

krishna619 commented:
@beingnoble03 Hi,
please find the SS 
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/52276473/652abf48-edaf-4efe-973c-9ce21b80e9f9)

beingnoble03 commented:
the above heading still says "User Name", can you make that right? 

maybe adding another column "Email" can fix it?

not sure if this would be the right way to do that? @aashimawadhwa what should be the right way to do that?

beingnoble03 commented:
@krishna619 I guess the simplest way would be to just replace "User Name" with "User", rest looks good.

beingnoble03 commented:
Add translations as well

krishna619 commented:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/52276473/9c9a2370-f7b1-4611-88c1-5653816cabeb)
@beingnoble03 I have changed the column name and added translations as well.
If you and @aashimawadhwa feel like we need a separate column for email in future you can let me now i'll do the needful.

krishna619 commented:
@palisadoes please review.

beingnoble03 commented on src/screens/OrganizationPeople/AddMember.tsx:
```diff
@@ -347,9 +348,7 @@ function AddMember(): JSX.Element {
-                      <StyledTableCell align="center">
-                        User Name
-                      </StyledTableCell>
+                      <StyledTableCell align="center">User</StyledTableCell>
```
@krishna619 the `User` translation is still missing

krishna619 replied:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/52276473/fed43385-977d-41ef-b352-636a6b77bf1c)
@beingnoble03 
added translation for user and add member , please review.

beingnoble03 reviewed (approved):
lgtm
